sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -13,18 +13,18 @@ namespace Azure.AI.TextAnalytics.Tests
     public class TextAnalyticsClientLiveTests : RecordedTestBase
     {
         public const string EndpointEnvironmentVariable = "TEXT_ANALYTICS_ENDPOINT";
-        public const string SubscriptionKeyEnvironmentVariable = "TEXT_ANALYTICS_SUBSCRIPTION_KEY";
+        public const string ApiKeyEnvironmentVariable = "TEXT_ANALYTICS_API_KEY";
```

Comment:
@chidozieononiwu currently the CI uses `TEXT_ANALYTICS_SUBSCRIPTION_KEY` but I am changing it to `TEXT_ANALYTICS_API_KEY`.
What do I have to do to make sure the CI works?

Reply:
You might need to change this here: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/test-resources.json#L84

Reply:
Nvm. Saw your change in the end of PR.

Reply:
@mssfang !! Yeah, I wasn't sure about it. 
Thanks for checking :)